typespec-extension/src/code-model-builder.ts

```diff
@@ -207,6 +207,8 @@ export class CodeModelBuilder {
       this.options["skip-special-headers"].forEach((it) => SPECIAL_HEADER_NAMES.add(it.toLowerCase()));
     }
 
+    this.options["group-etag-headers"] = this.options["group-etag-headers"] ?? true;
```

Comment:
We do not check `arm`?

Reply:
In the long term I think we don't. 
For green fields, before we fix etag headers grouping, we'll have to set this property to false in each tspconfig.yaml. After the fix, we can leave it out for green fields.
For brown fields, we'll always set it false.

Alternatively, as a short-term solution, we could do, if it's ARM and the flag is not set, we set it false. Actually, after a second thought, this one is quite appealing too, as we can selectively enable it for SDKs that really need it.

What's your opinion on this?

Reply:
I am OK either way. Anyway, at present, if tsp has it, arm can only use false, so it just automatically be false, or you need to set it to false.